common-lib/src/cmr/common/api/errors.clj

```diff
@@ -27,8 +27,8 @@
    :body {:errors ["An Internal Error has occurred."]}})
 
 (defn- keyword-path->string-path
-  "Converts a set of keyword field paths into the string equivalent field paths to return to the
-  user."
+  "Converts a set of keyword field paths into the string equivalent field paths
```

Comment:
Ah, I didn't realize that was a CMR thing. Bikeshed will complain about anything over 80, but not kibit. I was making the changes based on community standards/best practices ...

Is it okay to leave what has been done?

I'll make sure not to change long lines in the future. 

(Though my stuff will all be wrapped around 80; works better for me in terminals and with my split-panel editors -- I hope that's okay?)